.github/workflows/test.yml

```diff
@@ -238,7 +238,7 @@ jobs:
                 "\" }"),
 
               # uncomment this line when we need to depend on the dev version of libR-sys
-              'libR-sys = { git = "https://github.com/extendr/libR-sys" }',
+              'libR-sys = { git = "https://github.com/extendr/libR-sys", features = array("non-api", 1) }',
 
               sep = ";")
           
```

Comment:
Set here also `REXTENDR_SKIP_DEV_TESTS=TRUE`. Once https://github.com/extendr/rextendr/pull/352 is merged, it should work.